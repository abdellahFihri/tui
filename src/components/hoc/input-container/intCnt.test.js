import React from 'react';
import {shallow} from 'enzyme'
import InputContainer from './input-container'

it('expect to render inputContainer component',()=>{

    expect(shallow(<InputContainer/>)).toMatchSnapshot();
})