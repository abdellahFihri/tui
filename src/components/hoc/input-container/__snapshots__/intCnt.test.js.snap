// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render inputContainer component 1`] = `ShallowWrapper {}`;
